### PR TITLE
Added flask-redis recipe

### PR DESCRIPTION
**Pull request by DanielBok**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,82 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
-
```
Comment (marcelotrevisani) [inline recipes/example/meta.yaml]: Please, do not touch this file.
Could you please revert it?

File: recipes/flask-redis/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - flask_redis
+
+about:
+  home: https://github.com/underyx/flask-redis/
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
```
Comment (marcelotrevisani) [inline recipes/flask-redis/meta.yaml]: Please, delete the comments

File: recipes/example/meta.yaml (hunk 1 above)
Comment (DanielBok) [reply recipes/example/meta.yaml]: No problem. Rebased the commit to include it.

File: recipes/flask-redis/meta.yaml (hunk 2 above)
Comment (DanielBok) [reply recipes/flask-redis/meta.yaml]: Okay. removed the comments

**Pull request merged**